### PR TITLE
cice5: explicitly set the rpath of library dependencies

### LINKED ISSUE #14: cice5: shared libraries not found
```
$ ldd opt/spack/linux-rocky8-x86_64/intel-2019.5.281/cice5-master-byagur66qd4y7za7vauskh5w34t3cb54/bin/cice_auscom_1440x1080_48x40_480p.exe | grep not
	libnetcdf.so.18 => not found
	libnetcdff.so.7 => not found
```

### PR DESCRIPTION
**Pull request by harshula**
* Print the generated config file to stdout.
* Close #14.

### CODE REVIEW

File: packages/cice5/package.py [hunk 1]
```diff
@@ -63,6 +63,24 @@ class Cice5(MakefilePackage):
...
+    def get_linker_args(self, spec, name):
+        return " ".join(
+                    [(spec[name].libs).ld_flags,
+                    "-Wl,-rpath=" + join_path(spec[name].prefix, "lib")]
+                   )
+
+    # The reason for the explicit -rpath is:
+    # https://github.com/ACCESS-NRI/spack_packages/issues/14#issuecomment-1653651447
+    def make_linker_args(self, spec, name, namespecs):
+        path = join_path(spec["parallelio"].prefix, "lib")
```
Comment (aidanheerdegen) [inline packages/cice5/package.py:77]: Should this be
```suggestion
        path = join_path(spec[name].prefix, "lib")
```

**Review by aidanheerdegen — CHANGES_REQUESTED** [review]
I think just that one change required.

File: packages/cice5/package.py (hunk 1 above)
Comment (harshula) [reply packages/cice5/package.py:77]: Yes, that's a bug! Good catch.

**Review by aidanheerdegen — APPROVED** [review]
Looks good to me. If you had the output of the linker flags handy in a buffer somewhere it would be nice to paste it into the PR to show how all that logic pans out in an example build, but don't bother running it just to get that.

**Comment by harshula** [comment]
Good idea. I have it in my notes, so I've added it to the issue where the rest of the documentation resides.

**Pull request merged**